MekHQ/data/scenariotemplates/Prevent Recon.xml

```diff
@@ -48,6 +50,7 @@
                 <generationOrder>1</generationOrder>
                 <maxWeightClass>4</maxWeightClass>
                 <minWeightClass>0</minWeightClass>
+                <objectiveLinkedForces/>
```

Comment:
Why is this added as empty?

Reply:
No idea, that's just how the xml serializer renders it.